ros_buildfarm/common.py

```diff
@@ -184,7 +184,7 @@ def get_ci_job_name(rosdistro_name, os_name, os_code_name, arch, job_type):
 
 
 def get_ci_view_name(rosdistro_name):
-    view_name = '%sci' % rosdistro_name[0].upper()
+    view_name = '%sci' % rosdistro_name[:1].upper()
```

Comment:
This is a subtlety that might be worth a comment perhaps?

`''[0]` is an IndexError whereas `''[:1]` is `''`.

Reply:
Sure, 26f037d